data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -0,0 +1,441 @@
+# Python imports
+import re
+import argparse
+import logging
+
+# Third party imports
+import pandas as pd
+
+# Project imports
+from utils import bq
+import bq_utils
```

Comment:
Why are you using both the new `bq` module and the old `bq_utils` module?  We should prefer the new `bq` module.

Reply:
There are a couple of functions that are in `bq_utils` that do not exist yet in `bq`, I have them listed below. I will also need to adjust or add in an additional function to return job results. Currently `bq.query` returns a dataframe, while `bq_utils.query` returns `bq_service.jobs()`. 

- `bq_utils.wait_on_jobs`
- `bq_utils.BigQueryJobWaitError`

Should these be additional tickets, or should I just make these adjustments/additions to `bq` in this PR?

Reply:
I think those functions will be dropped entirely from the `utils/bq.py` module.  You can do the same thing with the existing client library functionality.
```
client = bq.get_client(project_id)
response = self.client.query('Select count(*) from `project.dataset.observation`')

# wait for the result using client library functions
results = response.result()

# check for an exception
exc = response.exception()
if exc:
    raise exc
```
This will wait on the job to finish automatically.  You can raise the exception if the job didn't finish correctly.  Here are some links to help.

https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.client.Client.html#google.cloud.bigquery.client.Client.query

https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.job.QueryJob.html#google.cloud.bigquery.job.QueryJob.result

Reply:
great, thank you for this, i was able to adjust!